src/yaml-support/yaml-util.ts

```diff
@@ -122,4 +122,39 @@ function getStringValue(node, key: string, ignoreCase: StringComparison = String
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// copied from explainer.ts, used by explainer.ts, yaml-schema.ts
```

Comment:
As these have been moved, you don't really need to describe the history -- that's what git logs are for.  (If you were duplicating them, then a TODO note to reunify the two copies would be useful - but I think you have kept a single implementation which is much better.)